### PR TITLE
WPB-11101 remove invitation tables from brig

### PR DESCRIPTION
**Pull request by battermann**
https://wearezeta.atlassian.net/browse/WPB-11101

## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

File: cassandra-schema.cql [hunk 1]
```diff
@@ -910,27 +891,6 @@ CREATE TABLE brig_test.connection (
     AND speculative_retry = '99PERCENTILE';
 CREATE INDEX conn_status ON brig_test.connection (status);
 
-CREATE TABLE brig_test.password_reset (
```
[the remark this answers is not in the document]
Comment (battermann) [reply cassandra-schema.cql:913]: IDK the diff looks very confusing

**Comment by battermann** [comment]
> > Don’t delete columns that are still used by versions that are deployed. If you delete columns then the old version will fail in the deployment process. Rather than deleting keep the unused columns around and comment them as being discontinued in the schema migration code.
> 
> https://docs.wire.com/developer/developer/pr-guidelines.html#schema-migrations

These tables are not used anywhere in our code

**Review by fisx — APPROVED** [review]

File: cassandra-schema.cql (hunk 1 above)
[the remark this answers is not in the document]
Comment (fisx) [reply cassandra-schema.cql:913]: for some reason the order is not stable. but it's auto-generated and tested, so you don't need to read it.

**Comment by fisx** [comment]
> > Don’t delete columns that are still used by versions that are deployed. If you delete columns then the old version will fail in the deployment process. Rather than deleting keep the unused columns around and comment them as being discontinued in the schema migration code.
> 
> https://docs.wire.com/developer/developer/pr-guidelines.html#schema-migrations

we're aware, but in this case we do not use the tables in *any* version, so all instances no older than the introduction of v0 should be fine.

**Comment by fisx** [comment]
@smatting sorry, didn't see your objection, let me know if you want this to be reverted on develop.

but i think we're fine...?

**Pull request merged**